src/modules/getArticles.js

```diff
@@ -1,8 +1,19 @@
 import axios from "axios";
 
-const getArticles = async () => {
-  const response = await axios.get("/articles", {});
-  return response.data.articles;
+const getArticles = {
+  async index() {
+    let result = await axios.get("/articles");
+    return result.data.articles;
+  },
+
+  async show(articleId) {
+    try {
+      let response = await axios.get(`/articles/${articleId}`);
+      return response.data.article;
+    } catch (error) {
+      return error.response.data.message;
+    }
+  },
 };
 
-export { getArticles };
\ No newline at end of file
+export default getArticles;
```

Comment:
So modules should actually be exported in brackets, not default, so change that back 👍🏻